x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -73,6 +73,8 @@ static RoleDescriptor kibanaSystem(String name) {
                 // For Fleet package upgrade
                 "manage_pipeline",
                 "manage_ilm",
+                // For connectors telemetry
+                "monitor_connector_state",
```

Comment:
__question:__ do I understand correctly this new privilege will eventually replace existing `RoleDescriptor.IndicesPrivileges.builder().indices(".elastic-connectors*").privileges("read")` index privilege? If so, do we have any issue tracking this cleanup work to maintain the `kibana_system` in good and current shape?

Reply:
You do understand that correctly. We'll be moving `.elastic-connectors*` to be system indices, so I'm pretty sure we'll be forced to clean this up.

Reply:
> You do understand that correctly. We'll be moving .elastic-connectors* to be system indices,

Okay, thanks for confirming.

> so I'm pretty sure we'll be forced to clean this up.

I hope so, but I'm not sure. Let's record this somewhere so we don't forget (context: we're trying really hard to contain what `kibana_system` is allowed to do).